### PR TITLE
adding cloudinary

### PR DESCRIPTION
**Pull request by escapekid**
Adding cloudinary setup 
pending to show view 
index is done

### CODE REVIEW

File: app/views/caravans/index.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
       <% @caravans.each do |caravan| %>
       <div class="col-3">
         <div class="card h-100">
-          <div class="card-image" style="background-image: url(https://images.unsplash.com/photo-1597192535374-aeba30156218?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80)">
+          <div class="card-image" style= "background-image: url(<%= cl_image_path(caravan.photo.key) if caravan.photo.attached? %>)">
```
Comment (Akki-on-Rails) [inline app/views/caravans/index.html.erb]: do we want to add a 'fallback' image in case the user doesn't upload a photo?

Comment (Jordan-Patel) [reply app/views/caravans/index.html.erb]: Yeah, great idea.

Comment (Akki-on-Rails) [reply app/views/caravans/index.html.erb]: @escapekid can you add a  else statement with a default image?

**Review by Akki-on-Rails — CHANGES_REQUESTED** [review]
please add a 'fallback' image.

**Comment by escapekid** [comment]
Adding fallback option - default image

**Review by Akki-on-Rails — APPROVED** [review]

File: app/views/caravans/index.html.erb (hunk 1 above)
Comment (menakadevisundaramoorthy) [reply app/views/caravans/index.html.erb]: ok

**Review by Akki-on-Rails — APPROVED** [review]

**Pull request merged**